project/FrgaalJavaCompiler.scala

```diff
@@ -26,7 +26,8 @@ import java.io.FileWriter
 object FrgaalJavaCompiler {
   private val ENSO_SOURCES = ".enso-sources"
 
-  val frgaal = "org.frgaal" % "compiler" % "19.0.0" % "provided"
+  val frgaal      = "org.frgaal" % "compiler" % "19.0.1" % "provided"
```

Comment:
There is a [version 20.0.0](https://central.sonatype.com/artifact/org.frgaal/compiler/20.0.0) out, but it warns about `ThreadDeath` being deprecated in JDK 20. As such we are upgrading just to 19.0.1, right?